### PR TITLE
Search f blimit

### PR DESCRIPTION
**Pull request by Rick-Methot-NOAA**
This augmentation will provide output of the equilibrium F associated with Blimit.  See issue #188

### CODE REVIEW

File: SS_benchfore.tpl [hunk 1]
```diff
@@ -980,6 +980,118 @@ FUNCTION void Get_Benchmarks(const int show_MSY)
...
+      {report5<<" SSB_Area:"<<p<<"_GP:"<<gp;}
+      report5<<endl;
+    }
+
+    F1(1)=log(1.0e-3); last_calc=0.; Fchange=-4.0; df=1.e-5; Closer=1.;
+    dvariable Closer2;
+    if(SR_fxn==5) {Closer2=0.001; Nloops=40;} else {Closer2=0.10; Nloops=28;}
+
+//    Btgttgt=BTGT_target*SSB_virgin;   //  this is relative to virgin, not to the average biology from benchmark years
+    Btgttgt2=0.5*Bmsy;   //  now relative to Bmark
```
Comment (chantelwetzel-noaa) [inline SS_benchfore.tpl]: Should the 0.5 here be the Blim_frac?

**Comment by k-doering-NOAA** [comment]
@Rick-Methot-NOAA , the build-warnings job is failing because there are new warnings this code is causing. I compared the artifact created by this run (the text file containing the warnings) with one from the main branch. The new warnings are: 
```
ss.cpp:28165:38: warning: statement has no effect [-Wunused-value]
28165 |     if(Do_Benchmark==3) {SPRloop1_end==8;}
      |                          ~~~~~~~~~~~~^~~
ss.cpp:28167:20: warning: statement has no effect [-Wunused-value]
28167 |       {SPRloop1_end==7;}
      |        ~~~~~~~~~~~~^~~

28215 |           Fmult2=Btgt_Fmult2;
      |                  ^~~~~~~~~~~
ss.cpp:28221:18: warning: implicitly-declared ‘dvariable& dvariable::operator=(const dvariable&)’ is deprecated [-Wdeprecated-copy]

ss.cpp: In member function ‘void model_parameters::SPR_profile()’:
ss.cpp:28172:34: warning: ‘SPRloop1_end’ may be used uninitialized in this function [-Wmaybe-uninitialized]
28172 |     for (int SPRloop1=0; SPRloop1<=SPRloop1_end; SPRloop1++)
      |  
```

Maybe the first two should be addressed? I think the third one can be safely ignored and the last one can be as well as long as you are confident SPRloop1_end is initialized. Let me what you decide to do, and I can modify the github action to accept them if we think they should be accepted

**Comment by Rick-Methot-NOAA** [comment]
some sloppy coding there.  Thanks for catching.

**Comment by k-doering-NOAA** [comment]
Great work, Rick, now there is only 1 new compiler warning instead of 4:
```
ss.cpp:28215:18: warning: implicitly-declared ‘dvariable& dvariable::operator=(const dvariable&)’ is deprecated [-Wdeprecated-copy]
28215 |           Fmult2=Btgt_Fmult2;
      |                  ^~~~~~~~~~~
```

Just checking, this warning is benign, right? If so, I will edit the gh actions job so that it doesn't flag this warning.

**Comment by Rick-Methot-NOAA** [comment]
That is correct.

I believe this may be because Btgt_Fmult2 is declared in PARAMETER_SECTION
as a number, and numbers declared there are considered by ADMB to be
variables; then Fmult2 is declared explicitly as a variable later in
FUNCTION SPR_profile.

**Comment by k-doering-NOAA** [comment]
@Rick-Methot-NOAA, the check has now been updated, so it passes! I support merging this into main, as long as Chantel agrees.

**Comment by chantelwetzel-noaa** [comment]
Thank you for doing the checks @k-doering-NOAA. Rick has addressed the only issue that I saw in review.  I think this is ready to be merged into the main branch.

**Pull request merged**